From 12a7d1e3beb313d2a3c4f22e70056af776e65666 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 6 May 2024 14:10:53 +0200
Subject: GCC: missing template instanciation in CSS parsing utils

In shorthands_custom.cc method ConsumeFont, the call to
ConsumeFontStretchKeywordOnly requires a template instanciation that
did not exist.

Bug: 40565911
Change-Id: Ic89b65e537dbfa5c1a14bdb86969315dda6829ce
---
 .../blink/renderer/core/css/properties/css_parsing_utils.cc    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/third_party/blink/renderer/core/css/properties/css_parsing_utils.cc b/third_party/blink/renderer/core/css/properties/css_parsing_utils.cc
index d16413b3470c4..a9d890cb971ea 100644
--- a/third_party/blink/renderer/core/css/properties/css_parsing_utils.cc
+++ b/third_party/blink/renderer/core/css/properties/css_parsing_utils.cc
@@ -6153,6 +6153,9 @@ CSSIdentifierValue* ConsumeFontStretchKeywordOnly(
 template CSSIdentifierValue* ConsumeFontStretchKeywordOnly(
     CSSParserTokenStream& stream,
     const CSSParserContext& context);
+template CSSIdentifierValue* ConsumeFontStretchKeywordOnly(
+    CSSParserTokenRange& stream,
+    const CSSParserContext& context);
 
 template <typename T>
   requires std::is_same_v<T, CSSParserTokenStream> ||
-- 
2.40.1

